doc/modules/neighbors.rst

```diff
@@ -464,6 +464,20 @@ leaf nodes.  The level of this switch can be specified with the parameter
 
 ``leaf_size`` is not referenced for brute force queries.
 
+Valid Metrics for Nearest Neighbor Algorithms
+---------------------------------------------
+
+For a list of available metrics, see the documentation of the :class:`DistanceMetric`
+class.
+
+ A list of valid metrics for any of the above algorithms can be obtained by using their
+``valid_metric`` attribute. For example, valid metrics for ``KDTree`` can be generated by:
```

Comment:
Getting a sphinx warning: 

`doc/modules/neighbors.rst:474: WARNING: Block quote ends without a blank line; unexpected unindent.`

Anyone knows how this can be fixed?